contents/data/sql/SQL.md

```diff
@@ -0,0 +1,145 @@
+After that, it groups those entries into the 3 courses: `CS101`, `CS202` and `CS404`.
+Then, it removes courses that do not have more than 1 student, i.e., `CS404` is removed from consideration.
+Finally, it returns a list of courses with a count of the number of students, ordered in increasing order.
+So, the output of the query is:
+
+| Course | num |
+| ------ | --- |
+| CS202  | 3   |
+| CS101  | 2   |
+
+<box type="info">
+  You can experiment with this example on <a href="https://www.db-fiddle.com/f/kHqV2edUGxCc1dU6vE6CmS/0">DB Fiddle</a> by entering SQL queries and then running them.
+</box>
+
+You can see how this simple query can prove to be extremely useful for getting this information when the table has tens of entries (or many more). Apart from retrieving information, SQL can also be used for creating, deleting and manipulating data with queries like  `INSERT`, `DELETE` and `UPDATE` for entries, in addition to `CREATE`, `DROP` and `ALTER` for tables as a whole.
+
+---
+
+## Why learn SQL?
+
+Now that you have seen how SQL is used, let's take a look at some of the main reasons behind the widespread adoption of SQL and why you should learn it.
+
+### Easy to learn
+
+As you saw in the example in the previous section, SQL is not really that complex. It is easy to learn, even for beginners who do not have any prior experience with databases. Since it has been around for a few decades, there are many good books and online resources to learn from.
+In addition, SQL (and SQL software) has a huge support community (such as [Stack Overflow](https://stackoverflow.com/questions/tagged/sql), and the [official MySQL forum](https://forums.mysql.com/)) which can prove useful when you run into problems while using it.
+
+### Free
+
+One of the benefits (and reasons) for SQL's popularity is that there is free software available (%like MySQL, PostgreSQL and SQLite%) as well as paid ones (%%like Microsoft SQL Server and Oracle%%) that come with more functionality.
+
+One such example is MySQL. It is free of cost and comes with the official [MySQL Workbench](https://dev.mysql.com/doc/workbench/en/) as well as other third party easy-to-use <tooltip content="Graphical User Interface">GUIs</tooltip>, which are less daunting to new users as compared to a <tooltip content="Command Line Interface">CLI</tooltip>.
+
+### Highly Compatible
+
+SQL software works on many operating systems (%%like Windows, Mac OS, Ubuntu, Red Hat among others%%) and more importantly, can be integrated with many languages. This includes programming languages like C++, Ruby, Java, Python, JavaScript (Node.js), and Go.
+
+In addition, online playgrounds like [DB Fiddle](https://www.db-fiddle.com/) and [JDoodle](https://www.jdoodle.com/execute-sql-online) make it easy to use and learn the SQL language quickly without the hassle of setting up an environment or application.
```

Comment:
Again, can show a screenshot  of a example to make things more visual. But not a must though.